.aws/src/event-rules/account-delete-monitor/index.ts

```diff
@@ -36,6 +42,13 @@ export class AccountDeleteMonitorEvents extends Resource {
     });
     this.createUserMergeRules();
     this.createPolicyForEventBridgeToSns();
+
+    createDeadLetterQueueAlarm(
+      this,
+      pagerDuty,
+      this.sqsDlq.name,
+      'AccountDeleteMonitorRule-Dlq-Alarm'
```

Comment:
can this prefix be replaced with a config var for proper grouping? like in the premium-purchase rule

Reply:
added prefix